actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+*   Add parameter filter capability for redirect locations.
+
+    It uses the `config.filter_parameters` to match what needs to be filtered.
+    The result would be like this:
+
+        Redirected to http://secret.foo.bar?username=roque&password=[FILTERED]
+
+    By security default, if no filter is provided, it does not filter any parameter.
```

Comment:
This sentence reads weird since we _don't filter any parameter_. Maybe it can be removed?